### PR TITLE
Use required-jcenter-modules-backup repository for Palantir modules

### PR DESCRIPTION
**Pull request by rpalcolea**
Hi @szimmer1 

We found a few projects that could be impacted by Bintray repos that are going away so we went ahead and created http://artifactory-oss.prod.netflix.net/artifactory/required-jcenter-modules-backup/ with these artifacts

Also, I removed the usages of `jcenter()`

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -44,6 +49,20 @@ subprojects {
 
     repositories {
         mavenCentral()
+        /**
+         * This repository locates artifacts that don't exist in maven central but we had to backup from jcenter or palantir repositories
+         * The exclusiveContent
```
Comment (szimmer1) [inline build.gradle:54]: Meant to finish this sentence?

**Comment by szimmer1** [comment]
@rpalcolea thanks for the PR and fixing my JAR hack :)

File: build.gradle (hunk 1 above)
Comment (rpalcolea) [reply build.gradle]: yeah, good catch! fixed it :)

**Comment by rpalcolea** [comment]
@szimmer1 fixed the broken sentence 👍 . This should be good now

**Pull request merged**